cli/tests/integrations/test_marathon.py

```diff
@@ -203,6 +203,17 @@ def test_remove_app():
     _list_apps()
 
 
+def test_add_multiapp():
+    with _zero_instance_multiapp():
+        _list_apps(['zero-instance-app1', 'zero-instance-app2'])
+
```

Comment:
Awesome, thanks! I implemented the one where both apps already exist. That is testing the new code that generates the multi line error message. The other cases are covered by the existing tests as they don't differ between single and multi app deployments. E.g. if one app exists but one has an error, from dcos-cli's perspective that's the same as a single app having an error as it's just sending the json object to Marathon without any regard for how many apps are in there. Like, there is no cli code that distinguishes at that point.